File: uri/C/beginner/1074.c
#include <stdio.h>

int main()
{

	int N, i, A;
	
	scanf("%d", &N);

	for(i = 1; i <= N; i++){
		scanf("%d", &A);
		if(A == 0){
		printf("NULL\n");
		}
		if(A < 0 && A % 2 ==0){
			printf("EVEN NEGATIVE\n");
		}
		if(A > 0 && A % 2 == 0){
			printf("EVEN POSITIVE\n");
		}
		if(A < 0 && A % 2 !=0){
			printf("ODD NEGATIVE\n");
		}
		if(A > 0 && A % 2 != 0){
			printf("ODD POSITIVE\n");
		}

	}

	return 0;
}

File: uri/python/1074.py
# -*- coding: utf-8 -*-

quantidade = int(input())
i = 1

while i <= quantidade:
	valor = int(input())
	if valor < 0 and valor % 2 == 0:
		print('EVEN NEGATIVE')
	if valor > 0 and valor % 2 == 0:
		print('EVEN POSITIVE')
	if valor < 0 and valor % 2 != 0:
		print('ODD NEGATIVE')
	if valor == 0:
		print('NULL')
	if valor > 0 and valor % 2 != 0:
		print('ODD POSITIVE')
	i += 1

